pkg/cloudprovider/provider/vsphere/types/cloudconfig.go

```diff
@@ -36,6 +36,7 @@ working-dir       = {{ .Global.WorkingDir | iniEscape }}
 datacenter        = {{ .Global.Datacenter | iniEscape }}
 datastore         = {{ .Global.DefaultDatastore | iniEscape }}
 server            = {{ .Global.VCenterIP | iniEscape }}
+cluster-id        = {{ .Global.ClusterID | iniEscape }}
```

Comment:
Where are you getting this value from ?

Reply:
The `global.ClusterID` is already in master. This PR is a follow-up of https://github.com/kubermatic/machine-controller/pull/1021/files.